.wordlist.txt

```diff
@@ -1,5 +1,6 @@
 # This wordlist is from the Sphinx starter pack and should not be
 # modified. Add any custom terms to .custom_wordlist.txt instead.
+# Leave a blank line at the end to support concatenation. 
```

Comment:
Maybe add the blank line as well? ;)

Reply:
Hmm, it does have one locally... Strange.

Reply:
Maybe you have Git configured to cut off blank lines at the end?